agent.go

```diff
@@ -28,6 +28,27 @@ type Agent struct {
 	Env       []string `json:"environments,omitempty"`
 }
 
+type AgentJobHistory struct {
+	Name                string                    `json:"job_name"`
+	JobStateTransitions []AgentJobStateTransition `json:"job_state_transitions"`
+	StageName           string                    `json:"stage_name"`
+	StageCounter        string                    `json:"stage_counter"`
```

Comment:
Not int?

Reply:
Nvm, looks consistently like a string in their API docs (oddly enough).

I must've been thinking about `original_job_id`:
https://github.com/ashwanthkumar/go-gocd/issues/18

Reply:
Yeah, just GoCD doing GoCD things...